src/fmt/mod.rs

```diff
@@ -41,13 +58,17 @@ use log::Record;
 
 #[cfg(feature = "humantime")]
 mod humantime;
+#[cfg(feature = "unstable-kv")]
+mod kv;
 pub(crate) mod writer;
 
 #[cfg(feature = "color")]
 pub use anstyle as style;
 
 #[cfg(feature = "humantime")]
 pub use self::humantime::Timestamp;
+#[cfg(feature = "unstable-kv")]
+pub use self::kv::*;
```

Comment:
If you are already going back to make a change, can you you check this on `RUSTDOCFLAGS=--cfg=docsrs cargo +nightly doc --all-features` (I'm hoping that is correct).

Each API item behind a `cfg` should have a note in the documentation, like [`clap::command` requiring `cargo`](https://docs.rs/clap/latest/clap/macro.command.html).

Most likely, you'll need to either duplicate this `cfg` on the items or move the `cfg` into the `kv` mod.

Reply:
Looks like it works as is:

![image](https://github.com/rust-cli/env_logger/assets/2541726/4eed76a1-4bfd-4f38-a7b3-5c9d46f8586c)

![image](https://github.com/rust-cli/env_logger/assets/2541726/7ec8a794-d53b-4fa7-93b0-dc6d21d6c84c)